storage/storageUnit/storageunit.go

```diff
@@ -139,6 +139,11 @@ func (u *Unit) PutInEpoch(key, data []byte, _ uint32) error {
 	return u.Put(key, data)
 }
 
+// GetOldestEpoch will return an error that signals that the oldest epoch fetching is not available
+func (u *Unit) GetOldestEpoch() (uint32, error) {
+	return 0, storage.ErrOldestEpochNotAvailable
```

Comment:
Wondering if we need to return an error here. Also valid for the nil storer. I think it would suffice to return the 0 and nil as to allow the oldDBCleaner instance to work properly and not output error messages.

Reply:
as discussed, I only added an error comparison in oldDatabaseCleaner, for logging reasons

Reply:
ok